Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp

```diff
@@ -174,19 +174,55 @@ namespace LegacyLevelSystem
             return false;
         }
 
+        // Make sure a spawnable level exists that matches levelname
+        AZStd::string validLevelName = "";
+        AZ::Data::AssetId rootSpawnableAssetId;
+        AZ::Data::AssetCatalogRequestBus::BroadcastResult(
+            rootSpawnableAssetId, &AZ::Data::AssetCatalogRequestBus::Events::GetAssetIdByPath, levelName, nullptr, false);
+
+        if (rootSpawnableAssetId.IsValid())
+        {
+            validLevelName = levelName;
+        }
+        else
+        {
+            // It's common for users to only provide the level name, but not the full asset path
+            // Example: "MyLevel" instead of "Levels/MyLevel/MyLevel.spawnable"
+            if (strstr(levelName, ".spawnable") == nullptr)
```

Comment:
This should use AZ::IO::PathView for determing the extension `AZ::IO::PathView(levelName).Extension() == ".spawnable"`

Reply:
fixed!